src/state/CalculationLogics.ts

```diff
@@ -75,6 +75,20 @@ export function getDaysBeforePeriod(cycles: Cycle[]) {
     };
   }
 
+  const periodLength = cycles[0].periodLength;
+  const dayOfCycle = Number(getDayOfCycle(cycles));
+
+  if (dayOfCycle <= periodLength) {
+    return {
+      title: i18n.t("Today is the"),
```

Comment:
A lot of text. It breaks on the phone screen. Replace "Today is the" to "Period:";
"day of your period" to "day".  Then together it will look like "Period: 2nd day".

Reply:
We already have `Period today` and `Period in` without `:` at the end of the first line. I changed to `Period 2nd day` without `:`